### ISSUE TITLE
"TypeError: Cannot read property 'raw' of undefined"

### ISSUE THREAD

**Issue by skddc** [sockethub/sockethub-platform-irc#30]
I'm currently stuck with this error for joining channels (using the default preset in Hyperchannel, so nothing funky going on). I do have a second XMPP space/connection open at the same time, but that's about it.

![screenshot from 2017-06-16 10-26-47](https://user-images.githubusercontent.com/842/27237548-54834c7c-527e-11e7-9e84-106d8c5e9c96.png)

In the Sockethub logs I can see:

```
sockethub:core failed job: 51 TypeError: Cannot read property 'raw' of undefined +2ms
sockethub:core job [51] failed for socket wyBrjBQh_FX2L-wvAAAE +0ms
```

**Comment by skddc** [comment]
This is currently breaking https://hyperchannel.kosmos.org it seems.

**Comment by silverbucket** [comment]
should be fixed in milestone v3.0

**Comment by skddc** [comment]
Great, thanks!
